# LOOPING STATEMENTS EXERCISE

---------------

### 1.Print -10 to -1 using for loop

In [2]:
for i in range (-10, 0):
    print (i)

-10
-9
-8
-7
-6
-5
-4
-3
-2
-1


### 2.Using while loop print numbers 1 to 10

In [7]:
i = 1
while i in range(1, 11):
    print (i)
    i = i+1


1
2
3
4
5
6
7
8
9
10


### 3.Using for loop print first ten even numbers

In [9]:
for i in range (1,21):
    if i%2 == 0:
        print (i)
    

2
4
6
8
10
12
14
16
18
20


### 4.Using while loop generate the table of 5

In [13]:
n = 5
i = 1
while i in range(1,100):
    print("{}*{}=".format(n,i), n*i)
    i = i+1

    

5*1= 5
5*2= 10
5*3= 15
5*4= 20
5*5= 25
5*6= 30
5*7= 35
5*8= 40
5*9= 45
5*10= 50
5*11= 55
5*12= 60
5*13= 65
5*14= 70
5*15= 75
5*16= 80
5*17= 85
5*18= 90
5*19= 95
5*20= 100
5*21= 105
5*22= 110
5*23= 115
5*24= 120
5*25= 125
5*26= 130
5*27= 135
5*28= 140
5*29= 145
5*30= 150
5*31= 155
5*32= 160
5*33= 165
5*34= 170
5*35= 175
5*36= 180
5*37= 185
5*38= 190
5*39= 195
5*40= 200
5*41= 205
5*42= 210
5*43= 215
5*44= 220
5*45= 225
5*46= 230
5*47= 235
5*48= 240
5*49= 245
5*50= 250
5*51= 255
5*52= 260
5*53= 265
5*54= 270
5*55= 275
5*56= 280
5*57= 285
5*58= 290
5*59= 295
5*60= 300
5*61= 305
5*62= 310
5*63= 315
5*64= 320
5*65= 325
5*66= 330
5*67= 335
5*68= 340
5*69= 345
5*70= 350
5*71= 355
5*72= 360
5*73= 365
5*74= 370
5*75= 375
5*76= 380
5*77= 385
5*78= 390
5*79= 395
5*80= 400
5*81= 405
5*82= 410
5*83= 415
5*84= 420
5*85= 425
5*86= 430
5*87= 435
5*88= 440
5*89= 445
5*90= 450
5*91= 455
5*92= 460
5*93= 465
5*94= 470
5*95= 475
5*96= 480
5*97= 485
5*98= 490
5*99= 495


### 5.Using for loop generate the table of 8

In [14]:
n = 8 
for i in range(1,100):
    print("{}*{}=".format(i,n), i*n)


1*8= 8
2*8= 16
3*8= 24
4*8= 32
5*8= 40
6*8= 48
7*8= 56
8*8= 64
9*8= 72
10*8= 80
11*8= 88
12*8= 96
13*8= 104
14*8= 112
15*8= 120
16*8= 128
17*8= 136
18*8= 144
19*8= 152
20*8= 160
21*8= 168
22*8= 176
23*8= 184
24*8= 192
25*8= 200
26*8= 208
27*8= 216
28*8= 224
29*8= 232
30*8= 240
31*8= 248
32*8= 256
33*8= 264
34*8= 272
35*8= 280
36*8= 288
37*8= 296
38*8= 304
39*8= 312
40*8= 320
41*8= 328
42*8= 336
43*8= 344
44*8= 352
45*8= 360
46*8= 368
47*8= 376
48*8= 384
49*8= 392
50*8= 400
51*8= 408
52*8= 416
53*8= 424
54*8= 432
55*8= 440
56*8= 448
57*8= 456
58*8= 464
59*8= 472
60*8= 480
61*8= 488
62*8= 496
63*8= 504
64*8= 512
65*8= 520
66*8= 528
67*8= 536
68*8= 544
69*8= 552
70*8= 560
71*8= 568
72*8= 576
73*8= 584
74*8= 592
75*8= 600
76*8= 608
77*8= 616
78*8= 624
79*8= 632
80*8= 640
81*8= 648
82*8= 656
83*8= 664
84*8= 672
85*8= 680
86*8= 688
87*8= 696
88*8= 704
89*8= 712
90*8= 720
91*8= 728
92*8= 736
93*8= 744
94*8= 752
95*8= 760
96*8= 768
97*8= 776
98*8= 784
99*8= 792


### 6. For numbers 1 to 100, if a number is divisible by 3, then print Fizz.
### If a number is divisible by 5, then print Buzz.
### If a number is divisible by 3 and 5 both, then print FizzBuzz
### If a number is neither divisible by 3 or 5, just print the number.

Hint: use range(1,101) function  to generate numbers from 1 to 100

In [15]:
for i in range(1,101):
    if (i%5 == 0 and i%3 == 0):
        print("FizBuzz")
    elif i%3 == 0:
        print("Fizz")
    elif i%5 == 0:
        print("Buzz")
    else:
        print(i)


1
2
Fizz
4
Buzz
Fizz
7
8
Fizz
Buzz
11
Fizz
13
14
FizBuzz
16
17
Fizz
19
Buzz
Fizz
22
23
Fizz
Buzz
26
Fizz
28
29
FizBuzz
31
32
Fizz
34
Buzz
Fizz
37
38
Fizz
Buzz
41
Fizz
43
44
FizBuzz
46
47
Fizz
49
Buzz
Fizz
52
53
Fizz
Buzz
56
Fizz
58
59
FizBuzz
61
62
Fizz
64
Buzz
Fizz
67
68
Fizz
Buzz
71
Fizz
73
74
FizBuzz
76
77
Fizz
79
Buzz
Fizz
82
83
Fizz
Buzz
86
Fizz
88
89
FizBuzz
91
92
Fizz
94
Buzz
Fizz
97
98
Fizz
Buzz


### 7.  Using the range function find the sum of all numbers from 1 to 100.
    For the numbers in the range (1,100):1+2+3+4+5......+100 = 5050

### 8. Write a program which will find all such numbers which are divisible by 7 but are not a multiple of 5, between 202 and 320 (both included).

In [16]:
for i in range(202, 321):
    if i%7 == 0 and i%5 != 0:
        print(i)

203
217
224
231
238
252
259
266
273
287
294
301
308


### 9. Python Program to count the number of vowels in a string.
st = "Data Science"

Output: No of vowels = 5

In [19]:
st = "Data Science"
count = 0
vowels = ["a", "e", "i", "o", "u"]
for i in st:
    if i in vowels:
        count = count+1
print("Output: No of vowels =", count)

Output: No of vowels = 5


### 10. Print Data Science 5 times using for loop

In [20]:
st = "Data Science"
for i in range(0,5):
    print(st)

Data Science
Data Science
Data Science
Data Science
Data Science
